build-system/tasks/runtime-test.js

```diff
@@ -105,11 +105,64 @@ function getAdTypes() {
   return adTypes;
 }
 
+
+/**
+ * Prints help messages for args if tests are being run for local development.
+ */
+function printArgvMessages() {
+  const argvMessages = {
+    safari: 'Running tests on Safari.',
+    firefox: 'Running tests on Firefox.',
+    edge: 'Running tests on Edge.',
+    saucelabs: 'Running tests on Sauce Labs.',
+    nobuild: 'Skipping build.',
+    watch: 'Enabling watch mode. Editing and saving a file will cause the' +
+        ' tests for that file to be re-run in the same browser instance.',
+    verbose: 'Enabling verbose mode. Expect lots of output!',
+    testnames: 'Listing the names of all tests being run.',
+    files: 'Running tests in the file(s): ' + util.colors.cyan(argv.files),
```

Comment:
Nit: Having some global constants for the colors may be slightly more readable.

```js
const cyan = utils.colors.cyan;
```

https://github.com/chalk/chalk#colors